src/main/java/ch/njol/skript/aliases/MaterialName.java

```diff
@@ -1,67 +1,66 @@
-/**
- *   This file is part of Skript.
+/*

```

Comment:
Aliases rework here too.

Reply:
What about that, should I revert any changes? I guess it still can be merged easily

Reply:
This will cause a merge conflict, which I can solve in about 5 seconds. The point was, don't spend time on these classes even if you decide to do more refactorings in other places. It would not be useful.